OWML.ModHelper/ModInputHandler.cs

```diff
@@ -0,0 +1,447 @@
+﻿using System;
+using System.Reflection;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using OWML.Common;
+using UnityEngine;
+
+namespace OWML.ModHelper
+{
+	public class ModInputHandler:IModInputHandler
```

Comment:
I think it makes sense for ModInputHandler and ModInputCombination to be in a new project called OWML.ModHelper.Input.

You can check @misternebula's PR, where he's adding a new project: https://github.com/amazingalek/owml/pull/93

Reply:
done